tests/unit/StaffNotation.spec.ts

```diff
@@ -10,13 +10,27 @@ Vue.use(Vuetify)
 const vuetify = new Vuetify({})
 const store = getVuexStore() as any // remove typings for unit tests
 
+const staffFilingTypes = [
+  {
+    name: 'Registrar\'s Notation',
+    type: 'registrarsNotation'
+  },
+  {
+    name: 'Registrar\'s Order',
+    type: 'registrarsOrder'
+  },
+  {
+    name: 'Court Order',
+    type: 'courtOrder'
+  }
+]
 describe('StaffNotation', () => {
   // Boilerplate to prevent the complaint "[Vuetify] Unable to locate target [data-app]"
   document.body.setAttribute('data-app', 'true')
   document.body.setAttribute('id', 'staff-notation')
-
+  store.state.isFirm = true
```

Comment:
`isFirm` is a getter, not a state.

Option 1: override this getter to return True.
Option 2: set the current entity type SP or GP, so the getter returns True.

Reply:
I changed it to store.state.entityType = 'SP'